### DISCUSSION TITLE
eBUS Adapter v3.1

### DISCUSSION THREAD

**Discussion by kratz00** [john30/ebusd#692] [Q&A] [answered]
I would like to know what has changed compared to v3.0? The only difference I spotted is the PCB. What else changed?

**Comment by john30** [comment, marked as answer]
it is functionally equivalent to 3.0, see the [documentation](https://adapter.ebusd.eu/index.en.html), just a bit of different chip set addressing the awful chip availability. and it was made simpler wrt usage

**Comment by kratz00** [comment]
Thanks for explaining it @john30
